lib/senkyoshi/models/questions/matching.rb

```diff
@@ -27,17 +30,20 @@ def iterate_xml(data)
           flow_label = response.at("flow_label")
           flow_label.children.each_with_index do |label, index|
             if label.attributes["ident"].value == answer_id
-              answer = matches_array[index]
+              answer = Nokogiri::HTML(matches_array[index]).text
```

Comment:
For these types of questions it was still showing the p tags if you don't take them out.